backend/app/models.py

```diff
@@ -1345,13 +1363,50 @@ class ModerationRequest(Base):
     comment = mapped_column(String)
 
 
+class GuidelineCategory(Base):
+    """A category of quality guidelines for an app"""
+
+    __tablename__ = "guideline_category"
```

Comment:
This is kind of against the naming of the other tables, but the previous naming scheme makes everything way too hard to read.